.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+  source = serenata_toolbox
+  omit =
```

Comment:
>  I would just like to quickly remind that the `*/virtualenv/*` rule was used because of the libraries included by Travis during the build process

Wow! Haven't noticed it yet — really good point!